examples/music-player/src/components/WelcomeScreen.tsx

```diff
@@ -1,32 +1,21 @@
 import { usePasskeyAuth } from "jazz-tools/react";
 import { ProfileForm } from "./ProfileForm";
 import { Button } from "./ui/button";
-import { useAccountSelector } from "@/components/AccountProvider.tsx";
+import {
+  useAccountSelector,
+  useAccountRef,
+} from "@/components/AccountProvider.tsx";
 
 export function WelcomeScreen() {
   const auth = usePasskeyAuth({
     appName: "Jazz Music Player",
   });
 
-  const { handleCompleteSetup } = useAccountSelector({
-    select: (me) =>
-      me.$isLoaded
-        ? {
-            id: me.root.$jazz.id,
-            handleCompleteSetup: () => {
-              me.root.$jazz.set("accountSetupCompleted", true);
-            },
-          }
-        : { id: null, handleCompleteSetup: null },
-    equalityFn: (a, b) => a.id === b.id,
```

Comment:
In my opinion, clearest example of why refs are useful here. It was quite convoluted to "expose" these handler functions without triggering unnecessary re-renders.